build/deploy.js

```diff
@@ -24,20 +24,30 @@ const deployToFirebase = (cb) => {
     return
   }
 
+  debug('Installing firebase-tools...')
+
+  exec('npm install -g firebase-tools', (error, stdout) => {
```

Comment:
Yeah, I see you there for sure. We just need to make sure the async flow is correct if doing it this way. Like you said though it should be a non-issue with Travis Firebase deploy.